launcher/internal/oci/interface.go

```diff
@@ -0,0 +1,59 @@
+// Package oci contains functionalities to interact with OCI image signatures.
+// https://github.com/opencontainers/image-spec/tree/main#readme.
+package oci
+
+import (
+	"go.uber.org/multierr"
+)
+
+// SigningAlgorithm is a specific type for string constants used for sigature signing and verification.
+type SigningAlgorithm string
```

Comment:
Do you want to support only this set of signing algorithms? Users may provide KMS keys with another algorithm. For example, here are all the supported GCP KMS algorithms - https://github.com/sigstore/sigstore/blob/main/pkg/signature/kms/gcp/client.go#L51-L62

Reply:
Yes, only RSA_SSA, RSA_PKCS1, and ECDSA_P256 will be supported, see `signing algorithm support` section on the design go/signed-image-impl-design